src/Services/Air/AirFormat.js

```diff
@@ -283,57 +284,103 @@ function formatLowFaresSearch(searchRequest, searchResult) {
 
   const fares = [];
 
-  Object.entries(pricesList).forEach(([fareKey, price]) => {
+  const isSolutionResult = typeof solutionsList !== 'undefined';
+
+  const results = isSolutionResult ? solutionsList : pricesList;
```

Comment:
Is that a BC change? If so, please bump the minor version in pkg.json
Not pretty sure what's the difference between solution/price, could you clarify that in the docs?

Reply:
solution returns only one schedule combination per pricing option per item, which is used in common nowadays, while AirPricePoint returns all available schedule per pricing options.

* [AirPricingSolutions](https://support.travelport.com/webhelp/uapi/Content/Air/Low_Fare_Shopping/Low_Fare_Shopping_(Synchronous).htm#AirPricingSolutions)

* [AirPricePointList]( https://support.travelport.com/webhelp/uapi/Content/Air/Low_Fare_Shopping/Low_Fare_Shopping_by_Price_Points.htm)